src/test/java/com/iota/iri/benchmarks/BenchmarkRunner.java

```diff
@@ -0,0 +1,31 @@
+package com.iota.iri.benchmarks;
+
+import com.iota.iri.benchmarks.dbbenchmark.RocksDbBenchmark;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import java.util.concurrent.TimeUnit;
+
+public class BenchmarkRunner {
+
+    @Test
+    public void launchDbBenchmarks() throws RunnerException {
+        Options opts = new OptionsBuilder()
+                .include(RocksDbBenchmark.class.getName() + ".*")
+                .mode(Mode.AverageTime)
+                .timeUnit(TimeUnit.MILLISECONDS)
+                .warmupIterations(5)
+                .forks(1)
```

Comment:
official docs recommend more forks. But since the test is already extremely long and I didn't see those forks help very much, I decreased it to 1.